crates/vault-registry/src/lib.rs

```diff
@@ -676,6 +824,68 @@ impl<T: Config> Module<T> {
         Ok(())
     }
 
+    /// Cancels a replace - which in the normal case decreases the old-vault's
+    /// to-be-redeemed tokens, and the new-vault's to-be-issued tokens.
+    /// When one or both of the vaults have been liquidated, this function also
+    /// updates the liquidation vault.
+    ///
+    /// # Arguments
+    /// * `old_vault_id` - the id of the old vault
+    /// * `new_vault_id` - the id of the new vault
+    /// * `tokens` - the amount of tokens to be transferred from the old to the new vault
+    pub fn cancel_replace_tokens(
+        old_vault_id: &T::AccountId,
+        new_vault_id: &T::AccountId,
+        tokens: PolkaBTC<T>,
+    ) -> DispatchResult {
+        Self::check_parachain_not_shutdown_and_not_errors(
+            [
+                ErrorCode::InvalidBTCRelay,
+                ErrorCode::OracleOffline,
+                ErrorCode::Liquidation,
```

Comment:
Maybe we can do this in another PR, but we should remove this ErrorCode since it's no longer set.

Reply:
made a ticket for this